object-browser.md

```diff
@@ -0,0 +1,68 @@
+```hbs
+{{frost-object-browser-lts
+  items=sortedItems
+  header=(component 'frost-info-bar'
+    icon=(component 'frost-icon'
+      icon='list-medium'
+      pack='frost'
+    )
+    title=(component 'text-box'
+      isVisible=isTitleVisible
+      text='&lt;placeholder: title&gt;'
+    )
+    summary=(component 'text-box'
+      isVisible=isSummaryVisible
+      text='&lt;placeholder: summary&gt;'
+    )
+    scope=(component 'text-box'
+      isVisible=isScopeVisible
+      text='&lt;placeholder: controls&gt;'
+    )
+    controls=(component 'frost-button'
```

Comment:
`{{component controls}}` is the current rendering code inside `info-bar`, I will address the change.